src/ast/Expr.zig

```diff
@@ -96,8 +96,8 @@ pub fn fromBlob(
 
     if (mime_type.category.isTextLike()) {
         var output = MutableString.initEmpty(allocator);
-        output = try JSPrinter.quoteForJSON(bytes, output, true);
-        var list = output.toOwnedSlice();
+        try JSPrinter.quoteForJSON(bytes, &output, true);
+        var list = output.slice();
```

Comment:
Same thing here, although it's not clear to me whether this actually ever gets freed